modules/ROOT/pages/add-api-to-unified-schema.adoc

```diff
@@ -119,9 +125,24 @@ To edit the authentication method:
 . Select an authentication policy and complete any required fields.
 . Click *Save*.
 
+[[configure-mtls-for-an-api-added-to-the-unified-schema]]
+== Configure mTLS for an API Added to the Unified Schema
+
+If you need to configure mTLS for an API that you've added to the unified schema, you can do so on the *API details* page.
+
+. Click *List of APIs added* and select the API that has the URL you want to edit.
+. Click *API details*.
+. .. Select *Configure Keystore* and provide the following:
+* A public certificate signed by your organization's “root” certificate authority (CA)
+* The private key used to generate the certificate signing request (CSR)
+* The password for the encrypted private key
+.. Optionally, select *Configure Truststore* and provide a link to the root CA.
```

Comment:
Same as with the add api. I would also add that the user can remove both the keystore and the truststore if the api had previously added ones. however the user will not be able to modify the keystore and truststore per se, it will need to remove the current keystore/truststore and add a new one. it can be done within the same action though